crud-client-go/client-go-crud/deployment.go

```diff
@@ -0,0 +1,94 @@
+package clientgocrud
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/apps/v1"
+	"k8s.io/client-go/util/retry"
+)
+
+func CreateDeployment(deploymentsClient v1.DeploymentInterface, clientset *kubernetes.Clientset) {
```

Comment:
CreateDeployment func should take the deployment object as args so that any deployment can be created. Just for knowledge, hardcoding here is fine

Reply:
Client generally you define in init() func so that it is available to the entire module and its not calling functions responsibility to provide the client to the DeploymentCreate() func

Reply:
These comments are applicable for each func in this assignment